### PR TITLE
Fix blockchain hook error prints

### PR DESCRIPTION
**Pull request by mariusmihaic**
Replaced `log.LogIfError` -> `log.Debug` in `blockchainHook.go`

### CODE REVIEW

File: process/smartContract/hooks/blockChainHook.go [hunk 1]
```diff
@@ -613,7 +613,10 @@ func (bh *BlockChainHookImpl) SetCurrentHeader(hdr data.HeaderHandler) {
 func (bh *BlockChainHookImpl) SaveCompiledCode(codeHash []byte, code []byte) {
 	bh.compiledScPool.Put(codeHash, code, len(code))
 	err := bh.compiledScStorage.Put(codeHash, code)
-	log.LogIfError(err, "func", "BlockChainHookImpl.SaveCompiledCode: compiledScStorage.Put", "codeHash", codeHash)
+	if err != nil {
+		log.Debug("BlockChainHookImpl.SaveCompiledCode: compiledScStorage.Put",
+			"err", err, "codeHash", codeHash)
```
Comment (bogdan-rosianu) [inline process/smartContract/hooks/blockChainHook.go:618]: `"err"` -> `"error"` ? applicable for the entire PR

**Review by sstanculeanu — DISMISSED** [review]

**Review by iulianpascalau — COMMENTED** [review]
What Bogdan found

File: process/smartContract/hooks/blockChainHook.go (hunk 1 above)
Comment (mariusmihaic) [reply process/smartContract/hooks/blockChainHook.go]: Right, fixed it.

**Review by sstanculeanu — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed.

**Pull request merged**